docs/guides/monitor/kubernetes-k8s-netdata.md

```diff
@@ -18,261 +31,223 @@ customization, or integration with your preferred alerting methods.
-discovery plugin, Kubelet collector, kube-proxy collector, and cgroups collector.
-
-To get started, open your browser and navigate to your cluster's Netdata dashboard. See our [Kubernetes installation
-instructions](/packaging/installer/methods/kubernetes.md) for how to access the dashboard based on your cluster's
-configuration.
-
-You'll see metrics from the parent pod as soon as you navigate to the dashboard:
+Running `kubectl get pods` shows both the Netdata and robot-shop deployments.
 
-![The Netdata dashboard when monitoring a Kubernetes
-cluster](https://user-images.githubusercontent.com/1153921/85343043-c6206400-b4a0-11ea-8de6-cf2c6837c456.png)
+```bash
+kubectl get pods --all-namespaces 
+NAMESPACE     NAME                              READY   STATUS    RESTARTS   AGE
+default       netdata-child-29f9c               2/2     Running   0          10m
+default       netdata-child-8xphf               2/2     Running   0          10m
+default       netdata-child-jdvds               2/2     Running   0          11m
+default       netdata-parent-554c755b7d-qzrx4   1/1     Running   0          11m
+kube-system   aws-node-jnjv8                    1/1     Running   0          17m
+kube-system   aws-node-svzdb                    1/1     Running   0          17m
+kube-system   aws-node-ts6n2                    1/1     Running   0          17m
+kube-system   coredns-559b5db75d-f58hp          1/1     Running   0          22h
+kube-system   coredns-559b5db75d-tkzj2          1/1     Running   0          22h
+kube-system   kube-proxy-9p9cd                  1/1     Running   0          17m
+kube-system   kube-proxy-lt9ss                  1/1     Running   0          17m
+kube-system   kube-proxy-n75t9                  1/1     Running   0          17m
+robot-shop    cart-b4bbc8fff-t57js              1/1     Running   0          14m
+robot-shop    catalogue-8b5f66c98-mr85z         1/1     Running   0          14m
+robot-shop    dispatch-67d955c7d8-lnr44         1/1     Running   0          14m
+robot-shop    mongodb-7f65d86c-dsslc            1/1     Running   0          14m
+robot-shop    mysql-764c4c5fc7-kkbnf            1/1     Running   0          14m
+robot-shop    payment-67c87cb7d-5krxv           1/1     Running   0          14m
+robot-shop    rabbitmq-5bb66bb6c9-6xr5b         1/1     Running   0          14m
+robot-shop    ratings-94fd9c75b-42wvh           1/1     Running   0          14m
+robot-shop    redis-0                           0/1     Pending   0          14m
+robot-shop    shipping-7d69cb88b-w7hpj          1/1     Running   0          14m
+robot-shop    user-79c445b44b-hwnm9             1/1     Running   0          14m
+robot-shop    web-8bb887476-lkcjx               1/1     Running   0          14m
+```
 
-Remember that the parent pod is responsible for storing metrics from all the child pods and sending alarms.
+## Explore Netdata's Kubernetes monitoring charts
```

Comment:
Users are used to seeing the agent dashboard streamed over to the cloud. Perhaps it's worth to add a sentence explicitly stating that the new viz is only available to the cloud dashboard overview tab.

Reply:
I feel like it's already fairly-well explained here. The deployment doc, which they would have already read through, also explains the location of the new features.